src/handlers/assign.rs

```diff
@@ -508,7 +510,10 @@ pub(super) async fn handle_command(
                 {
                     // This is a comment, so there must already be a reviewer assigned. No need to assign anyone else.
                     issue
-                        .post_comment(&ctx.github, &on_vacation_msg(&username))
```

Comment:
the deletion of `on_vacation_msg` is in 82c549a7b397079b73449dc4e4e181da2e6b174b (sorry)

Reply:
I find the functions better than the consts, because with the function you cannot forget to pass the required placeholders, with the static string you can easily just use it without replacing the placeholder.